docker/.env.example

```diff
@@ -0,0 +1,26 @@
+# Secrets
+# YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+JWT_SECRET=your-secret
+
+## General
+NODE_ENV=local
+MONGO_URL=mongodb://mongodb:27017/notifire-db
+REDIS_HOST=redis
+
+# AWS
+S3_LOCAL_STACK=http://localhost:4566
+S3_BUCKET_NAME=notifire-local
+S3_REGION=us-east-1
+AWS_ACCESS_KEY_ID=test
+AWS_SECRET_ACCESS_KEY=test
+
+# Ports
+API_PORT=3000
+REDIS_PORT=6379
+WS_PORT=3002
+
+# Root URL
+REACT_APP_API_URL=http://localhost:3000
+REACT_APP_WS_URL=http://localhost:3002
```

Comment:
A small one here, can we remove the REACT_APP part and only use API_URL (We can combine it with API_ROOT_URL and map it in the docker-compose-file to "REACT_APP_*) @djabarovgeorge

Reply:
Absolutely can be done! :)